graylog2-web-interface/src/components/streams/Stream.jsx

```diff
@@ -113,6 +114,7 @@ class Stream extends React.Component {
     this.setState({ showEntityShareModal: true });
   };
 
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
we should move the function outside of the Class and that should also fix this linter issue.